frontend/app/dtos/parse-address-request-dto.server.ts

```diff
@@ -0,0 +1,33 @@
+export interface ParseWSAddressRequestDTOProps {
+  addressLine: string;
+  city: string;
+  province: string;
+  postalCode: string;
+  country: string;
+  language: string;
+}
+
+export class ParseWSAddressRequestDTO implements ParseWSAddressRequestDTOProps {
+
+  addressLine: string;
+  city: string;
+  province: string;
+  postalCode: string;
+  country: string;
+  language: string;
+
+  constructor(addressLine: string, 
+    city: string,
+    province: string,
+    postalCode: string,
+    country: string,
+    language: string) {
+
+    this.addressLine = addressLine;
+    this.city = city;
+    this.province = province;
+    this.postalCode = postalCode;
+    this.country = country;
+    this.language = language;
+  }
+}
```

Comment:
In typescript, this class is unnecessary since you can simply do this to create a data object:

``` typescript
const address: ParseWSAddressRequestDTOProps = {
  addressLine: '..',
  city: '..',
  province: '..',
  postalCode: '..',
  country: '..',
  language: '..',
}
```